system/utils.py

```diff
@@ -248,7 +376,7 @@ def wrapped(operations, queue_size, add_size, get_size, event_loop):
                                                                                                   -len(':0')]
     except TypeError:
         try:
-            time.sleep(120)
+            await asyncio.sleep(120)
```

Comment:
I do understand that it was here originally, but:
- it looks like this whole thing can be replaced with `eventually`
- I suspect retry timeout can be reduced to 60-70 seconds